### ISSUE TITLE
Error loading the Projects for Explore Projects

### ISSUE THREAD

**Issue by frafra** [hotosm/tasking-manager#4065]
This happens when opening `/manage/projects/?status=PUBLISHED&managedByMe=1`.

```
2021-01-07 17:23:41,790 CRITICAL: Project GET - unhandled error: {"action": ["Value must be one of ('map', 'validate', 'any')."]} [in /usr/src/app/backend/api/projects/resources.py:671]
```

Version 4.2.3.

**Comment by frafra** [comment]
I think that catching the errors without displaying the stack trace is not a good practice, because it hides the real issue:

```
        except Exception as e:
            error_msg = f"Project GET - unhandled error: {str(e)}"
            current_app.logger.critical(error_msg)
            return {"Error": "Unable to fetch projects"}, 500
```

The tasking manager has plenty of those generic`except` statements.

**Comment by frafra** [comment]
Here is the full traceback:

```
Traceback (most recent call last):
  File "/usr/src/app/backend/api/projects/resources.py", line 664, in get
    search_dto = self.setup_search_dto()
  File "/usr/local/lib/python3.7/site-packages/flask_httpauth.py", line 163, in decorated
    return f(*args, **kwargs)
  File "/usr/src/app/backend/api/projects/resources.py", line 533, in setup_search_dto
    search_dto.validate()
  File "/usr/local/lib/python3.7/site-packages/schematics/models.py", line 258, in validate
    partial=partial, convert=convert, app_data=app_data, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/schematics/models.py", line 299, in _convert
    return func(self._schema, self, raw_data=raw_data, oo=True, context=context, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/schematics/validate.py", line 67, in validate
    raise DataError(errors, data)
schematics.exceptions.DataError: {"action": ["Value must be one of ('map', 'validate', 'any')."]}
```

**Comment by frafra** [comment]
Our setup uses Italian as main language: https://osmit-tm4.wmcloud.org/
I have two PC with Firefox: one set to English, the other set to Italian. The one set to Italian seems to work fine, but the one set to English fails.

**Comment by willemarcel** [comment]
@frafra which version are you using? We fixed that 500 error on v4.2.4

**Comment by frafra** [comment]
@willemarcel as mentioned in the first post, I was using v4.2.3. It got fixed with v4.2.4.
Sorry for the bug report! I thought I was using the latest stable :) Thanks!
